.github/workflows/scripts/slack_notifications/slack_notifier.rb

```diff
@@ -0,0 +1,32 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require 'httparty'
+
+class SlackNotifier
+  CYCLE = 24 * 60 * 60 # Period in seconds to check for updates that need to be Slacked.
+  @@errors = []
```

Comment:
For future PRs:

- As discussed synchronously earlier, the need to go with a class variable has demonstrated that we should refactor the notifiers to use an instance of `SlackNotifier` and have `SlackNotifier`'s current static/class methods become dynamic/instance methods.
- Currently the model prevents us from running both notifiers at the same time, because they will share an errors array. That's enough of an indication to move to instance methods and an instance variable, `@errors`.